src/components/AddLinkInput/AddLinkInput.tsx

```diff
@@ -0,0 +1,56 @@
+import { CSSProperties, useEffect, useRef, useState } from "react";
+import styles from "./AddLinkInput.module.scss";
+import InputIcon from "@/assets/svg/InputIcon";
+
+export default function AddLinkInput() {
+  const [isAddLinkInputVisible, setAddLinkInputVisible] = useState(true);
+  const addLinkInputRef = useRef(null);
```

Comment:
DOM에 접근하는 용도로 useRef를 사용할때는 요렇게 써주는 편입니다~
제네릭 타입을 넣어주냐, 아니냐에 따라서 RefObject, MutableRefObject와 같이 타입이 달라져요!
```suggestion
  const addLinkInputRef = useRef<HTMLElement>(null);
```

Reply:
해당 사항에 대한 자료는 따로 찾아보고 다시 수정 진행해보겠습니다...!